integration-tests/integration/createIncident/form-submit.cy.js

```diff
@@ -23,7 +23,7 @@ context('Submit the incident report', () => {
     cy.task('stubUserDetailsRetrieval', ['MR_ZAGATO', 'MRS_JONES', 'TEST_USER'])
   })
 
-  it('Submitting a form', () => {
+  it.only('Submitting a form', () => {
```

Comment:
Do you still want this `.only` here or was it just a temporary thing?

Reply:
no it was a temp thing which I forgot to remove.